scripts/buildtools/install_buildtools.sh

```diff
@@ -93,3 +93,4 @@ install_go_module github.com/go-swagger/go-swagger github.com/go-swagger/go-swag
 install_go_module github.com/algorand/msgp
 install_go_module gotest.tools/gotestsum
 install_go_module github.com/algorand/oapi-codegen github.com/algorand/oapi-codegen/cmd/oapi-codegen
+install_go_module github.com/golangci/golangci-lint/cmd/golangci-lint
```

Comment:
this means everywhere we build we will now install a linter? i don't think that makes sense from logical separation.

Reply:
AFAIK this is how we control versions of our tools

Reply:
i mean that we might want to separate our build process and our setup for development process. We shouldn't need lint to build a binary.

Reply:
The CI calls `install_buildtools` with arguments for what it wants.

This change is to make development easier, currently `make lint` doesn't reflect warnings generated by CI.